src/views/add-member.html

```diff
@@ -4,117 +4,113 @@
   {% include "include/back-link.html" %}
 {% endblock %}
 
-{% block content %}
```

Comment:
Hi Nayyab,

The changes involved with this PR involve de-denting all the pages `html` code by to accommodate the implementation of the common width container.

I believe that `git` has detected the `add-member` changes as inline because it's a small file with no embedded `nunjucks` code (it mostly uses common components). Most of the other pages are much larger so I think it's detecting them as chunk based changes.